tests/test_dataset.py

```diff
@@ -110,3 +110,26 @@ def test_dataset_only_takes_common_filenames(tmp_path):
 
     assert dataset.image_paths == expected_image_paths
     assert dataset.grace_paths == expected_label_paths
+
+
+def test_gracedir_from_stardir(tmp_path):
+    star_fns = ["b", "a", "aj", "jj"]
+
+    star_fns = [f + ".png" for f in star_fns]
+
+    star_dir = tmp_path / "star"
+    star_dir.mkdir()
+
+    for fn in star_fns:
+        file = star_dir / fn
+        file.touch()
+
+    mkdir_grace_from_star(stardir=star_dir)
+
+    expected_grace_paths = [
+        tmp_path / "grace" / "b.grace",
+        tmp_path / "grace" / "jj.grace",
+    ]
+
+    assert expected_grace_paths[0].is_file() is True
```

Comment:
This would work equally with `assert expected_grace_paths[0].is_file()` since `is_file()` returns a bool anyway so we don't need `is True`